packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,416 @@
+  <Menu
+    itemKey="name"
+    items={[
+      {name: 'Copy', icon: 'Copy', shortcut: '⌘C'},
+      {name: 'Cut', icon: 'Cut', shortcut: '⌘X'},
+      {name: 'Paste', icon: 'Paste', shortcut: '⌘V'}
+    ]}>
+    {item => {
+      let iconMap = {
+        Copy,
+        Cut,
+        Paste
+      };
+      let Icon = iconMap[item.icon];
+      return (
+        <Item childItems={item.children} textValue={item.name}>
+          <Icon size="S" />
+          <Text>{item.name}</Text>
+          <Keyboard>{item.shortcut}</Keyboard>
+        </Item>
+      );
+    }}
+  </Menu>
+</MenuTrigger>
+```
+
+### Accessibility
+
+The MenuTrigger uses the prop `aria-controls` to point at the menu, which uses
+the prop `aria-labelledby` to reference the trigger. Other aria props of the
+MenuTrigger are `aria-haspopup` and `aria-expanded`.
+
+Menu sections without labels use the prop `aria-label`.
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Section 1', children: [{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}, {name: 'Section 2', children: [{name: 'Koala'}]}]} itemKey="name">
+    {item => (
+      <Section items={item.children} aria-label={item.name}>
+        {item => <Item>{item.name}</Item>}
+      </Section>
+    )}
+  </Menu>
+</MenuTrigger>
+```
+
+## Events
+
+MenuTrigger accepts an `onOpenChange` handler which is triggered whenever the Menu is opened or closed.
+
+The example below uses `onOpenChange` to update a separate span element with the current open state of the Menu.
+
+```tsx example
+function Example() {
+  let [state, setState] = React.useState(false);
+
+  return (
+    <div>
+      <MenuTrigger onOpenChange={(isOpen) => setState(isOpen)}>
+        <ActionButton>
+            Menu Button
+        </ActionButton>
+        <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+          {item => <Item>{item.name}</Item>}
+        </Menu>
+      </MenuTrigger>
+      <span style={{'margin-left': '8px'}}>Current open state: {state.toString()}</span>
+    </div>
+  );
+}
+```
+
+Menu accepts an `onAction` handler which is triggered whenever a Menu item is selected.
+
+The example below uses `onAction` to tell the user which Menu item was selected.
+
+```tsx example
+function Example() {
+  let onAction = (value) => alert('Menu item selected: ' + value);
+
+  return (
+    <div>
+      <MenuTrigger>
+        <ActionButton>
+            Menu Button
+        </ActionButton>
+        <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name" onAction={onAction}>
+          {item => <Item>{item.name}</Item>}
+        </Menu>
+      </MenuTrigger>
+    </div>
+  );
+}
+```
+
+## Props
+
+### MenuTrigger
+
+<PropTable component={docs.exports.MenuTrigger} links={docs.links} />
+
+### Menu
+
+<PropTable component={docs.exports.Menu} links={docs.links} />
+
+## Visual Options
+
+### Align and Direction
```

Comment:
Might be worthwhile naming each button to reflect what alignment and direction is being applied to make it easier on the user